### DISCUSSION TITLE
broadcasters

### DISCUSSION THREAD

**Discussion by m7zr** [TychoTheTaco/Twitch-Drops-Bot#65] [Ideas]
can we put more then one channel in the "broadcasters": [""] ?
ex : "broadcasters": ["hhhhhh,ggggggg,222222"]

**Comment by TychoTheTaco** [comment]
Yes, you can do it like this:
Using config file: `"broadcasters": ["aaa", "bbb", "ccc"]`
Using command line: `--broadcasters=aaa,bbb,ccc`
